src/programs/state/massachusetts/housing.js

```diff
@@ -3,11 +3,14 @@
 import { CHILD_CARE_EXPENSES } from '../../../data/state/massachusetts/name-cores';
 
 // UTILITIES
-import { toCashflow, sumCashflow, getGrossUnearnedIncomeMonthly } from '../../../helpers/cashflow';
-import { sum } from '../../../helpers/math';
+import { sum } from '../../../utils/math';
+import { Result } from '../../../utils/Result';
```

Comment:
1. This comment doesn't matter because we're going to change what calculations return.
2. `Result` seems to be to be in a different category than a utility and I'm not sure this way of writing it clarifies anything about its use in this context.

Reply:
Let me know what you want done with `Result`. I don't think it's enough of a special snowflake to get special treatment unless it truly represents a different category of logic than other utilities.

Reply:
We're gonna nuke it, but it'll be a different issue. This is just part of my journey of self discovery.

Reply:
:snowman_with_snow: